sdcm/cluster.py

```diff
@@ -1783,7 +1783,26 @@ def scylla_setup(self, disks):
         """
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         devname = result.stdout.strip()
-        self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {}'.format(devname, ','.join(disks)))
+        if self.parent_cluster.params.get('workaround_kernel_bug_for_iotune'):
+            # related issue: https://github.com/scylladb/scylla/issues/5181
+            # a known kernel bug will cause scylla_io_setup fails in executing iotune.
+            # The kernel bug doesn't occur all the time, so we can get some succeed gce instance.
+            # the config files are copied from a succeed gce instance (same instance type, same test).
+            self.remoter.run(
+                'sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} --no-io-setup'.format(devname, ','.join(disks)))
+            script = dedent("""echo \"disks:
```

Comment:
if you are using dedent indent the text so it will look nice, that's the whole point of dedent

Reply:
The indent will effect the output of echo, the generated config will be different.

Expected output:
```
disks:
  - mountpoint: /var/lib/scylla
    read_iops: 540317
    read_bandwidth: 1914761472
    write_iops: 300319
    write_bandwidth: 1162022400
```

The output with nice indent:
```
disks:
          - mountpoint: /var/lib/scylla
            read_iops: 540317
            read_bandwidth: 1914761472
            write_iops: 300319
            write_bandwidth: 1162022400
```

Reply:
Oh, I see now: this is YAML format. so the best way is to bring the file to the test machine open it with YAML parser,
edit and upload back. These text manipulations with echo, tee, pipes are error-prone

Reply:
Fixed. I saved the good config files to ./configurations/ directory.
And copy them to instances in setup, overwrite is fine.